pytest-backend/pyproject.toml

```diff
@@ -14,7 +14,7 @@ packages = [{include = "exasol"}]
 
 [tool.poetry.dependencies]
 pytest = ">=7,<9"
-exasol-integration-test-docker-environment = "^3.1.0"
+exasol-integration-test-docker-environment = "^4.1.0"
```

Comment:
did the itde api start api change frrom 3 to 4, if not we should allow both

Reply:
Yes, it changed, see https://github.com/exasol/pytest-plugins/pull/105/files#r2215474895